MySQLOO/source/GMModule.cpp

```diff
@@ -122,14 +125,35 @@ static int fetchFailed(lua_State* state) {
 static int doVersionCheck(lua_State* state) {
 	GarrysMod::Lua::ILuaBase* LUA = state->luabase;
 	LUA->SetState(state);
-	LUA->PushSpecial(GarrysMod::Lua::SPECIAL_GLOB);
-	LUA->GetField(-1, "http");
-	LUA->GetField(-1, "Fetch");
-	LUA->PushString("https://raw.githubusercontent.com/FredyH/MySQLOO/master/minorversion.txt");
-	LUA->PushCFunction(fetchSuccessful);
-	LUA->PushCFunction(fetchFailed);
-	LUA->Call(3, 0);
-	LUA->Pop(2);
+
+	// Check if the reference to the ConVar object is set
+	if (iVersionCheckConVar != NULL) {
+		LUA->PushSpecial(GarrysMod::Lua::SPECIAL_GLOB); // Push the global table

```

Comment:
This isn't needed as you aren't using the global table

Reply:
Accidently kept this in from when I was testing the code at the bottom of the module open function, removed now.